api/Document.json

```diff
@@ -5879,10 +5921,12 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/height",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "1",
+              "version_removed": true
```

Comment:
It was probably removed long ago, this is introducing uncertainty for what's in current Chrome. Can you either add the exact removal version or defer this change until later when it is known?